todo_app/my_ansible_playbook.yml

```diff
@@ -0,0 +1,50 @@
+- name: install and run to-do app
+  hosts: webservers
+  remote_user: ec2-user
+  vars_files:
+  - secrets.yml
+
+  tasks:
+  - name: install Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: latest
+  - name: install Python
+    become: yes
+    ansible.builtin.yum:
+        name: python3
+        state: latest
+  - name: install Poetry
+    ansible.builtin.shell: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+  - name: directory for To-Do App
+    ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        owner: ec2-user
+  - name: Checkout Git Repo
+    become: yes
+    ansible.builtin.git:
+        repo: https://github.com/Wizty79/DevOps-Course-Starter-02
+        dest: /opt/todoapp
+        version: exercise_04
+        clone: yes
+        update: yes
```

Comment:
Nothing wrong with including default arguments for clarity, but worth noting if you hadn't already that it should [behave the same](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html#parameter-update) even if you remove these